src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml

```diff
@@ -183,12 +175,10 @@ tests:
                 value: 2000
 
     ### Mandatory Depends on the FeatureMap
-    - label:
-          "3b: If (PA & LF) TH reads CurrentPositionLiftPercent100ths attribute
-          from DUT"
+    - label: "3b: TH reads CurrentPositionLiftPercent100ths attribute from DUT"
       command: "readAttribute"
       attribute: "CurrentPositionLiftPercent100ths"
-      PICS: WNCV.S.F00 && WNCV.S.F02 && WNCV.S.A000e
+      PICS: WNCV.S.A000e && ( WNCV.S.F00 || WNCV.S.F01 )
```

Comment:
wrong

Reply:
Removed changes from PR please verify it.